google-cloud-errorreporting/src/main/java/com/google/cloud/errorreporting/spi/v1beta1/ErrorStatsServiceSettings.java

```diff
@@ -319,14 +348,21 @@ private Builder(ErrorStatsServiceSettings settings) {
       listEventsSettings = settings.listEventsSettings.toBuilder();
       deleteEventsSettings = settings.deleteEventsSettings.toBuilder();
 
-      methodSettingsBuilders =
-          ImmutableList.<ApiCallSettings.Builder>of(
+      unaryMethodSettingsBuilders =
+          ImmutableList.<UnaryCallSettings.Builder>of(
               listGroupStatsSettings, listEventsSettings, deleteEventsSettings);
     }
 
+    private static ConnectionSettings.Builder s_getDefaultConnectionSettingsBuilder() {
```

Comment:
@garrettjonesgoogle Looks like we should fix the function names. I think Garrett is working on the refactor of these connection settings, so I suggest we can fix these with the refactoring (assuming this is a non-blocking issue).